code/modules/mob/living/say.dm

```diff
@@ -55,11 +55,6 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	"å" = "Syndicate",
 	"í" = "CentCom",
 
-	// Species
-	"è" = "binary",
-	"ï" = "changeling",
-	"ô" = "alientalk",
-

```

Comment:
???

Reply:
isn't need anymore due to saymode datums

Reply:
)))

Reply:
You're missing the other place this is set some lines above then

Reply:
either way it doesn't really matter

Reply:
do both or do neither

Reply:
k